app/javascript/components/pill.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class Pill extends React.Component {
+  render(){
+    const { option, url, text } = this.props;
+    const className = 'component-pill ' + option;
```

Comment:
@aSadhankar you can use string interpolation (not really necessary, but a nice option)

Take note of the backticks

```
const className = `component-pill ${option}`;
```